dependabot/dependabot.go

```diff
@@ -64,7 +64,7 @@ func checkRepo(ctx context.Context, client *github.Client, repo *github.Reposito
 	}()
 
 	// get content of circleci config file
-	content, SHA, err := gh.GetRepoContent(ctx, client, repoOwner, repoName, ".circleci/config.yml", targetBranch)
+	content, SHA, err := gh.GetRepoContent(ctx, client, repoOwner, repoName, repoConfig.Directory+"/.circleci/config.yml", targetBranch)
```

Comment:
Is this correct? Isnt this the config file for circleci and not dependabot-circleci?

Reply:
Nevermind... i just read the jira task.. hah :D

Reply:
It's soo annoying when reading stuff is required 🤣